src/Numbers/Numbers.js

```diff
@@ -4,14 +4,15 @@ export const Numbers = (props) => {
     numbers.push(i);
   }
   if (props.odd) {
-    numbers = numbers.filter(num => num % 2);
-  }
-  if (props.even) {
-    numbers.filter(num => !num % 2);
+    numbers = numbers.filter(num => num % 2 !== 0);
+  } else if (props.even) {
+    numbers = numbers.filter(num => num % 2 === 0);
+  } else if (!props.odd && !props.even) {
+    return null;
```

Comment:
why null? It should return the whole list if there are no _odd_ ore _even_ flags